distro/quarkus-docker-compose/.env.template

```diff
@@ -0,0 +1,68 @@
+#APICURIO VERSION
+APICURIO_API_VERSION=latest
+APICURIO_WS_VERSION=latest
+APICURIO_UI_VERSION=latest
+
+#HOSTNAME OR IP ADDRESS
+HOST=$HOST
+
+#KEYCLOAK
+
+KC_MYSQL_DATABASE=keycloak
+KC_MYSQL_ROOT_PASSWORD=$KC_ROOT_DB_PASSWORD
+KC_MYSQL_USER=keycloak
+KC_MYSQL_PASSWORD=$KC_DB_PASSWORD
+
+KEYCLOAK_USER=admin
+KEYCLOAK_PASSWORD=$KC_PASSWORD
+
+DB_VENDOR=mysql
+DB_ADDR=jboss-keycloak-mysql
+DB_USER=keycloak
+DB_PASSWORD=$KC_DB_PASSWORD
+
+QUARKUS_PROFILE=prod
+
+#MICROCKS
+
+MR_SPRING_PROFILES_ACTIVE=prod
+MR_SPRING_DATA_MONGODB_URI=mongodb://mongo:27017
+MR_SPRING_DATA_MONGODB_DATABASE=microcks
+MR_POSTMAN_RUNNER_URL=http://postman:3000
+MR_TEST_CALLBACK_URL=http://$HOST:8090
+MR_KEYCLOAK_URL=http://$HOST:8090/auth
+
+#APICURIO-STUDIO
+
+AS_DATABASE=apicuriodb
+AS_MYSQL_ROOT_PASSWORD=$AS_MYSQL_ROOT_PASSWORD
+AS_DB_USER=apicurio
+AS_DB_PASSWORD=$AS_DB_PASSWORD
+
+APICURIO_KC_AUTH_URL=http://$HOST:8090/auth/realms/apicurio
```

Comment:
I think this is incorrect, it should be APICURIO_KC_AUTH_URL=http://$HOST:8090/auth

Reply:
No, for the Quarkus app, the correct value for the auth URL has to be like `http://$HOST:$PORT/auth/realms/$REALM`.